src/object.c

```diff
@@ -613,10 +613,16 @@ int isObjectRepresentableAsLongLong(robj *o, long long *llval) {
  * string. This happens because SDS strings tend to overallocate to avoid
  * wasting too much time in allocations when appending to the string. */
 void trimStringObjectIfNeeded(robj *o) {
```

Comment:
looking at the previous PR, and the changes that didn't make it here, don't we wanna simplify the comment in tryObjectEncoding, and remove the outdated details from it?

Reply:
Done.